CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. Each pull request must be approved by 2 contributors (not including yourself).
```

Comment:
I agree with @erquhart.

As usual, check docker/docker for an awesome contributing guide:
https://github.com/moby/moby/blob/master/CONTRIBUTING.md